example/src/App.tsx

```diff
@@ -52,13 +54,18 @@ export default function App() {
     <VideoroomContextProvider>
       <NavigationContainer linking={linking}>
         <Stack.Navigator
-          initialRouteName="Home"
+          initialRouteName="InitialScreen"
           screenOptions={{
-            headerShown: false,
+            headerBackTitle: 'Back',
           }}
         >
-          <Stack.Screen name="Home" component={Home} />
+          <Stack.Screen
+            name="CreateRoom"
```

Comment:
Changing the name of the screen from `Home` to `CreateRoom` will break deep linking (see `linking` object above). Change the linking object and verify that deeplinking still works

Reply:
Done.